samples/apply-scene-property-expressions/README.md

```diff
@@ -0,0 +1,37 @@
+# Apply scene property expressions
+
+Update the orientation of a graphic using expressions based on its attributes.
+
+![Image of apply scene property expressions](apply-scene-property-expressions.png)
```

Comment:
The screenshot has a watermark and is quite squished.

What I'd recommend is to size your screenshot to a width of 700 pixels (and keep ratio for height), then grab a 400 pixel chunk out of the screenshot